pkg/monitor/monitor.go

```diff
@@ -241,7 +242,8 @@ func (m *Monitor) cleanupDriverNodePods(node *volume.NodeInfo) {
 	if err == nil {
 		return
 	}
-	pods, err := core.Instance().GetPods("", nil)
+
+	pods, err := storkcache.Instance().ListPods()
```

Comment:
Is it ideal to list Pods from cache where pods are the objects which are most likely change?

Reply:
Cache is always eventually consistent. However in this case we need to optimize the total space we are using to store Pod lists.